scripts/openapi-sdk/generate_openapi_client.sh

```diff
@@ -27,4 +27,5 @@ npx --yes @openapitools/openapi-generator-cli generate -g typescript-axios -i \
 "$openapi_filename" -o "$output_path" \
 --remove-operation-id-prefix \
 --package-name=$package_name \
---additional-properties=$additional_properties
+--additional-properties=$additional_properties \
+--type-mappings object=Object
```

Comment:
This change is important, so we get:
```
export interface Action {
    /**
     * 
     * @type {string}
     * @memberof Action
     */
    'type': string;
    /**
     * 
     * @type {{ [key: string]: Object; }}
     * @memberof Action
     */
    'parameters': { [key: string]: Object; };
}
```

instead of:
```
export interface Action {
    /**
     * 
     * @type {string}
     * @memberof Action
     */
    'type': string;
    /**
     * 
     * @type {{ [key: string]: object; }}
     * @memberof Action
     */
    'parameters': { [key: string]: object; };
}
```

Please notice `Object` vs `object`. This particular class, in java has field declaration like `Map<String, Object> parameters;`. So values in java can be strings. In typescript, `object` would force just non primitive values. String wouldn't be allowed.

Reply:
I see `Object` is avoiding us a lot of trouble here but I think we should avoid it. It's usually discouraged to use it as a type.

We could stay with `{ [key: string]: object; }`. But honestly I think it's harder to manage than having just `parameters: object`.

Ideally it should be like `{ [key: string]: unknown; }`, which is the closest to the actual thing inside `parameters`. but I'm not sure we have a way to generate it. So maybe we could just leave `parameters: object` and do type assertions where needed.

Also, taking a look around, many things that need to be fixed following the change to `parameters` could be removed after we switch to the dynamically generated forms (`ActionEdit`, `ProcessorDetail`, the validation hook etc.). 

So we have also the option to keep this PR on hold and update the SDK together with the changes to the processor action/source management.

Reply:
_So we have also the option to keep this PR on hold and update the SDK together with the changes to the processor action/source management._ what is the estimate for it @kelvah please? Or in other words, what jira blocks this?

Reply:
There are no blockers at the moment. I've started to look into it. Difficult to do an estimation though. I need some more time. Let's chat about it.